### ISSUE TITLE
preferences/verify-paths BATS test fails on macOS with 1.15.0 release build

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#7338]
This fails repeatedly on both macOS Sonoma (Intel) and Sequoia Beta (Arm):

```
 ✗ zsh unmanaged
   (from function `refute_output' in file bats-assert/src/refute_output.bash, line 189,
    in test file tests/preferences/verify-paths.bats, line 78)
     `refute_output --partial "$HOME/.rd/bin/rdctl"' failed

   -- output should not contain substring --
   substring : /Users/jan/.rd/bin/rdctl
   output    : /Users/jan/.rd/bin/rdctl
   --

 ✗ fish unmanaged
   (from function `refute_output' in file bats-assert/src/refute_output.bash, line 189,
    in test file tests/preferences/verify-paths.bats, line 87)
     `refute_output --partial "$HOME/.rd/bin/rdctl"' failed

   -- output should not contain substring --
   substring : /Users/jan/.rd/bin/rdctl
   output    : /Users/jan/.rd/bin/rdctl
   --
```

Or on a different machine:

```
 ✗ move to manual path-management [8545]
   (from function `try' in file tests/helpers/utils.bash, line 283,
    in test file tests/preferences/verify-paths.bats, line 62)
     `try --max 5 --delay 2 no_bashrc_path_manager' failed
   Status: settings updated; no restart required.

 ✗ bash unmanaged [243]
   (from function `refute_output' in file bats-assert/src/refute_output.bash, line 189,
    in test file tests/preferences/verify-paths.bats, line 69)
     `refute_output --partial "$HOME/.rd/bin/rdctl"' failed

   -- output should not contain substring --
   substring : /Users/jan/.rd/bin/rdctl
   output    : /Users/jan/.rd/bin/rdctl
   --

 ✗ zsh unmanaged [247]
   (from function `refute_output' in file bats-assert/src/refute_output.bash, line 189,
    in test file tests/preferences/verify-paths.bats, line 78)
     `refute_output --partial "$HOME/.rd/bin/rdctl"' failed

   -- output should not contain substring --
   substring : /Users/jan/.rd/bin/rdctl
   output    : /Users/jan/.rd/bin/rdctl
   --

 - fish unmanaged (skipped: fish not found) [9]
```

**Comment by mook-as** [comment]
This test seems to pass on my machine (1.15.0, macOS); there's likely an issue with some local files (probably `~/.bashrc` etc)?

**Comment by jandubois** [comment]
In manual testing I see the following diagnostic:

```
2024-08-29T15:44:38.553Z: Check LIMA_DARWIN_VERSION result: {"description":"This machine is running macOS 15.1.0.","passed":true,"fixes":[]}
2024-08-29T15:44:38.553Z: Check PATH_MANAGEMENT:.bash_profile result: {"description":"Failed to determine if `/Users/jan/.bash_profile` contains extended attributes","passed":false,"fixes":[],"id":".bash_profile"}
```

Maybe there is an issue with `fs-xattr` on macOS Sequoia, but unfortunately the error string doesn't include the text of the original exception.

**Comment by jandubois** [comment]
Fixed by #7397
